ansible/all-in-one/roles/xsnippet/vars/main.yml

```diff
@@ -0,0 +1,21 @@
+# software versions
+xsnippet_api_image: xsnippet/xsnippet-api:latest
+xsnippet_db_image: docker.io/library/postgres:13
+xsnippet_web_image: docker.io/library/nginx:stable
+xsnippet_web_assets: https://github.com/xsnippet/xsnippet-web/releases/download/v1.1.3/xsnippet-web-v1.1.3.tar.gz
+
+# application configuration
+xsnippet_syntaxes: "{{ lookup('file', 'syntaxes.txt').split('\n') }}"
+
+# reverse-proxy configuration
+xsnippet_api_server_name: api.xsnippet.org
+xsnippet_spa_server_name: xsnippet.org
+access_log_format: '%{X-Real-IP}i "%r" %s %b %{User-Agent}i" %Tf'
+
+# SSL/TLS configuration
+xsnippet_api_https_redirect: false  # do not force HTTPS redirect for now to allow for local development
```

Comment:
nit: I think it should be true with an option to override the value via `--extra-vars` upon ansible playbook invocation.

Reply:
Sounds good!